test/system/create_note_test.rb

```diff
@@ -24,4 +24,25 @@ class CreateNoteTest < ApplicationSystemTestCase
       end
     end
   end
+
+  test "can open new note page when zoomed out" do
```

Comment:
Wouldn't it make more sense to start zoomed in and then test that things disable when you zoom out? It's more realistic than starting from a zoomed out URL which is something the site would never offer as the control is still disabled at low zoom.

I mean there's nothing wrong with testing this, but we should test that zooming out triggers disablement as well.

Reply:
Starting zoomed out was a scenario from #1291 that I was going to fix. You can get there by opening `/note/new` from `/fixthemap`.

Added starting zoomed in, then zooming out, then zooming in again.